plugins/google-sheets/src/sheets.ts

```diff
@@ -294,9 +294,12 @@ function getFieldValue(fieldType: CollectionFieldType, cellValue: CellValue) {
         }
         case "date": {
             if (typeof cellValue !== "string") return null
-            const [month, day, year] = cellValue.split(/[-/]/).map(Number)
-            const date = new Date(Date.UTC(year, month - 1, day, 12, 0, 0, 0))
-            return date.toISOString()
+        try {
+                const date = new Date(Date.parse(cellValue))
+                return date.toISOString()
+            } catch (e) {
```

Comment:
Nit. We try not to use abbreviations. But e seems unused here so you can completely remove the variable I think